Src/Plugins/SensitiveTerms/SEC106.SensitiveTerms.json

```diff
@@ -0,0 +1,138 @@
+{
+  "ValidatorsAssemblyName": "SensitiveTerms.dll",
+  "SharedStringsFileName": "SensitiveTerms.SharedStrings.txt",
+  "Definitions": [
+    {
+      "Id": "SEC106/001",
+      "Name": "DoNotUseSensitiveTerms.Accessibility",
+      "Level": "Warning",
+      "FileNameAllowRegex": "$SourceFiles",
```

Comment:
>$SourceFiles [](http://example.com/codeflow?start=29&length=12)

Why are we restricting analysis only to source files? You should create a $TextFiles expression that contains all of source code but also things like txt, markdown, etc.